apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/DexController.java

```diff
@@ -537,65 +540,105 @@ public Response getContractForOrder(@Parameter(description = "APL account id (RS
+        return Response.ok( new TradingDataOutputUpdatedToDtoConverter().apply(tradingDataOutputUpdated) ) .build();
     }
-
+    
+    
     @GET
-    @Path("/histohour")
+    @Path("/symbols")
     @Produces(MediaType.APPLICATION_JSON)
-    @Operation(tags = {"dex"}, summary = "Get histohour", description = "getting histohour")
+    @Operation(tags = {"dex"}, summary = "Get history", description = "getting history")
     @ApiResponses(value = {
             @ApiResponse(responseCode = "200", description = "Exchange offers"),
             @ApiResponse(responseCode = "200", description = "Unexpected error") })
-    public Response getHistohour1(  @Parameter(description = "Type of exchange (coinbase)") @QueryParam("e") String e,
-                                @Parameter(description = "fsym") @QueryParam("fsym") String fsym,
-                                @Parameter(description = "tsym") @QueryParam("tsym") String tsym,                                
-                                @Parameter(description = "toTs") @QueryParam("toTs") Integer toTs,
-                                @Parameter(description = "limit") @QueryParam("limit") Integer limit,
+    public Response getSymbols(   @Parameter(description = "Cryptocurrency identifier") @QueryParam("symbol") String symbol,                                                            
                                 @Context HttpServletRequest req) throws NotFoundException {
 
-        log.debug("getHistohour:  fsym: {}, tsym: {}, toTs: {}, limit: {}", fsym, tsym, toTs, limit);            
-        TradingDataOutput tradingDataOutput = getDataForIntervalFromOffers(  fsym,  tsym,  toTs, limit,  60*60, service, timeService);
-        return Response.ok( new TradingDataOutputToDtoConverter().apply(tradingDataOutput) ) .build();
+        log.debug("getSymbols:  fsym: {}", symbol );
+        SymbolsOutputDTO symbolsOutputDTO = new SymbolsOutputDTO();
+        symbolsOutputDTO.name = symbol;
+        symbolsOutputDTO.exchange_traded = "Apollo Currency";
+        symbolsOutputDTO.exchange_listed = "Apollo Currency";
+        symbolsOutputDTO.timezone = "America/New_York";
+        symbolsOutputDTO.minmov = 1;
+        symbolsOutputDTO.minmov2 = 0;
+        symbolsOutputDTO.pointvalue = 1;
+        symbolsOutputDTO.session = "0930-1630";
+        symbolsOutputDTO.has_intraday = false;
+        symbolsOutputDTO.has_no_volume = false;
+        symbolsOutputDTO.description = "Apollo Foundation.";
+        symbolsOutputDTO.type = "stock";
+        symbolsOutputDTO.supported_resolutions = new ArrayList<>();;
+        symbolsOutputDTO.supported_resolutions.add("D");
+        symbolsOutputDTO.supported_resolutions.add("2D");
+        symbolsOutputDTO.supported_resolutions.add("3D");
+        symbolsOutputDTO.pricescale = 1000000000;
+        symbolsOutputDTO.ticker = symbol;
+        return Response.ok( symbolsOutputDTO ) .build();
     }
-
+    
+    
     @GET
-    @Path("/histoday")
+    @Path("/config")
```

Comment:
Maybe better to separate graph from root url '/dex' by adding subresource  /dex/graph

Reply:
It would be better to leave it as it is at the moment since it requires many changes from the side of the frontend.